Automattic/wp-calypso
# Sites Popover: Styling Migration

#### Changes proposed in this Pull Request

* Simple migration of the sites popover component

#### Testing instructions

This is used in two places, one of which is the "Write" button on the masterbar, so just click that and verify the styling is the same.

<img width="502" alt="Screenshot 2019-07-06 at 15 56 11" src="https://user-images.githubusercontent.com/43215253/60757860-01175680-a008-11e9-953e-5335165a9804.png">

The other is `/devdocs/blocks/daily-post-button`, but I'm wondering if the Daily Post Button block can just go? (Daily Post ended a while ago and I can't find it used in Calypso)

cc @blowery, @jsnajdr, @flootr 

Closes #34499
Part of #27515
